pkg/systemd/host.css

```diff
@@ -278,6 +278,12 @@ body {
     word-wrap: break-word;
 }
 
+#system_machine_id {
+    display: inline-block;
+    font-family: monospace;
+    word-break: break-all;
```

Comment:
Is word-wrapping really intended? I thought we wanted to ellipsize and show a tooltip, or not wrap and just make the  box wide enough and drop the tooltip?

Reply:
Answering myself: the wrapping makes sense for really narrow displays, and it looks ok. I'll drop the tooltip.